wom/src/test/scala/wom/types/WomTypeSpec.scala

```diff
@@ -0,0 +1,167 @@
+package wom.types
+
+import org.scalatest.prop.TableDrivenPropertyChecks._
+import org.scalatest.prop.Tables.Table
+import org.scalatest.{FlatSpec, Matchers}
+import spray.json.JsString
+import wom.values._
+
+import scala.runtime.ScalaRunTime
+
+class WomTypeSpec extends FlatSpec with Matchers {
+  "WdlType class" should "stringify WdlBoolean to 'Boolean'" in {
```

Comment:
Ahem...

Reply:
yeah I've fixed this and a ton of other stuff in subsequent commits, but there's sooo much WDL... 😢